operator/cmd/mesh/shared.go

```diff
@@ -47,6 +48,10 @@ import (
 var (
 	// installerScope is the scope for all commands in the mesh package.
 	installerScope = log.RegisterScope("installer", "installer", 0)
+
+	// testK8Interface is used if it is set. Not possible to inject due to cobra command boundary.
+	testK8Interface *kubernetes.Clientset
+	testRestConfig  *rest.Config
```

Comment:
Why we need the test client configs here?

Reply:
these are generated during specially during test startup and cannot be injected as noted in the comment.